lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties

```
@@ -199,7 +199,7 @@ digitizing.toolbar.polygon=Polygon
 digitizing.toolbar.freehand=Free hand
 digitizing.toolbar.import=Import in GeoJSON, GPX or KML
 digitizing.toolbar.erase=Delete all features
```

```suggestion
digitizing.toolbar.erase=Delete some features
```

The user is free to remove one or many with this label ?